src/main/kotlin/org/radarbase/redcap/managementportal/Subject.kt

```diff
@@ -41,7 +41,7 @@ data class Subject(
     @JsonProperty("login") val subjectId: String,
     @JsonProperty("externalId") val externalId: Int? = null,
     @JsonProperty("externalLink") val externalLink: URL? = null,
-    @JsonProperty("project") val project: Project,
+    @JsonProperty("project") var project: Project? = null,
```

Comment:
Ah yes when querying the project endpoint, it did not return the `Project` anymore. I updated the MP query to use the project endpoint because the subject endpoint was not working correctly.